platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/Migratable.java

```diff
@@ -37,15 +35,64 @@
  * </p>
  */
 public interface Migratable extends Describable {
+    /**
+     * Gets the current export version handled by this migratable.
+     * <p>
+     * When exporting data, the migration framework will export this version information and provide
+     * it back to the {@link #doImport} method as part of the import migration context when importing
+     * the corresponding exported data.
+     *
+     * @return the current version handled by this migratable
+     */
+    @Override
+    @Nullable
+    public String getVersion();
 
     /**
-     * Exports all migratable data to the specified directory
+     * Exports all required migratable data to the specified context.
+     * <p>
+     * Errors, warnings, and/or information messages can be recorded along with the context's report.
+     * Doing so will not abort the operation right away.
```

Comment:
✏️  If recording to the report never aborts the operation, should we remove _right away_ from the comment?